src/jqLite.js

```diff
@@ -519,29 +521,34 @@ function jqLiteDocumentLoaded(action, win) {
   }
 }
 
+function jqLiteReady(fn) {
+  var fired = false;
+
+  function trigger() {
+    if (fired) return;
+    fired = true;
+    fn();
+  }
+
+  // check if document is already loaded
+  if (window.document.readyState === 'complete') {
+    window.setTimeout(trigger);
+  } else {
+    // We can not use jqLite since we are not done loading and jQuery could be loaded later.
+
+    // Works for modern browsers and IE9
+    window.document.addEventListener('DOMContentLoaded', trigger);
+
+    // Fallback to window.onload for others
+    window.addEventListener('load', trigger);
```

Comment:
By using `addEventListener` (here and above) doesn't it mean the people can't use `.triggerHandler()`?
It could break apps (and even more likely tests).